float32 time 
float32 PrevPosX
float32 PrevPosY
float32 PrevPosZ
---
float32 PosX
float32 PosY
float32 PosZ
float32 VelX
float32 VelY
float32 VelZ
float32 AccX
float32 AccY
float32 AccZ
